apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -302,3 +306,7 @@ type IngressCapabilities struct {
 	IngressProvider      string `json:"ingressProvider,omitempty"`
 	CustomDefaultBackend bool   `json:"customDefaultBackend,omitempty"`
 }
+
+type MonitoringInput struct {
+	Answers map[string]string `json:"answers,omitempty"`
```

Comment:
What is the UX going to be around this? How are we going to present the questions and answers?

Reply:
This struct uses to overwrite the Answers of `cluster-monitoring` App and `project-monitoring` App, and we only allow users to overwrite some important fields, just like Prometheus retention, Storage Class, Storage Size and Node Exporter Port.